clientv3/watch.go

```diff
@@ -52,6 +59,22 @@ type WatchResponse struct {
 	Canceled bool
 }
 
+// IsCreateEvent returns true if the event tells that the key is newly created.
+func IsCreateEvent(e *Event) bool {
+	if e.Kv == nil {
+		return false
+	}
+	return e.Kv.CreateRevision == e.Kv.ModRevision
+}
+
+// IsModifyEvent returns true if the event tells that modification occured on the key.
+func IsModifyEvent(e *Event) bool {
+	if e.Kv == nil {
```

Comment:
Delete has a kv with an empty value. Otherwise, how would you know which key was deleted?